boards/slwstk6220a/Makefile.features

```diff
@@ -2,6 +2,5 @@ CPU = ezr32wg
 CPU_MODEL = ezr32wg330f256r60
```

Comment:
looks like `cpu/ezr32wg/Makefile.features` needs an update too.

Reply:
Nice catch. I've reworked the git history to not mix efm32 related changes with ezr32wg (but kept the latter anyway).